README.md

```diff
@@ -740,6 +740,48 @@ You should be able to pass data via `worker.workerData` (see [Custom Worker Opti
 
 Note that you cannot pass a built-in nor bound function.
 
+## Typescript Usage
+
+As is mentioned in this issue: https://github.com/breejs/bree/issues/24 and thanks to @ChrisEdgington.  You can use typescript workers as:
+
+Create a function that will process the path:
+
+```js
+function typescript_worker() {
+    const path = require('path')
+    require('ts-node').register()
+    const workerData = require('worker_threads').workerData
+    require(path.resolve(__dirname, workerData.__filename))
+}
+```
+
+Then use the the function as path (see [Using functions for jobs](#using-functions-for-jobs)), and pass the path to your ts job file:
```

Comment:
The CI are failing because the repeated "the"

Reply:
ouch! Fixed